snuba/cli/rust_consumer.py

```diff
@@ -140,6 +140,14 @@
     default=False,
     help="Enable async inserts for ClickHouse",
 )
+@click.option(
+    "--allow-mutability",
```

Comment:
probably rename this to something else (or ideally, really make a separate command

Reply:
separate command meaning a different entrypoint than rust_consumer?

Reply:
I would say in the short term we should rename this flag to something better, long term we need a completely separate CLI

Reply:
Renamed it, let me know what you think?